core/src/main/java/org/elasticsearch/script/Script.java

```diff
@@ -161,14 +166,22 @@ public String getLang() {
         return params;
     }
 
+    public XContentType getContentType() {
```

Comment:
totally agreed, it for mustache templates that have been defined as a json object instead of a plain string scripts. For instance: 

```
{
   "inline" : {
       "match_{{type}}" : {
           "{{field}}" : "{{value}}"
       }
   }
}
```